src/vue/components/login.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <button @click="logIn" class="uk-button uk-button-primary" v-show="! this.$store.state.auth.isAuthenticated">
+      {{ l10n.getMsg(`AUTH_LOGIN_BTN_LABEL`) }}
+    </button>
+    <button @click="logOut" class="uk-button uk-button-primary" v-show="this.$store.state.auth.isAuthenticated">
+      {{ l10n.getMsg(`AUTH_LOGOUT_BTN_LABEL`) }}
+    </button>
+  </div>
+</template>
+<script>
+
+export default {
+  name: 'UserAuth',
+  inject: {
+    l10n: 'l10n',
+    auth: { from: 'auth', default: null } // This module is options
+  },
+  data: function () {
```

Comment:
Probably we should add a `storeModules: ['auth']` here and add a dependency check mixin. This will guarantee that an `auth` store module is present

Reply:
e.g. after the `inject`

Reply:
done